portugues/stack/stack.md

```diff
@@ -0,0 +1 @@
+O stackoverflow é um fórum de perguntas e respostas que pode ajudar a solucionar o problema. Todo programador usa pelo menos uma vez por dia. Aqui está o [link](<https://stackoverflow.com/>).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.